Source/WebKit/WebProcess/Plugins/PDF/PDFPluginBase.mm

```diff
@@ -195,7 +258,12 @@
     memcpy(CFDataGetMutableBytePtr(m_data.get()) + m_streamedBytes, buffer.data(), buffer.size());
     m_streamedBytes += buffer.size();
 
-    incrementalPDFStreamDidReceiveData(buffer);
+#if !LOG_DISABLED
+    pdfLog(makeString("PDFPluginBase::streamDidReceiveData() - received ", buffer.size(), " bytes, total streamed bytes ", m_streamedBytes));
```

Comment:
What's with the bespoke logging thing. Is this normal? And the fact that it depends on makeString instead of itself being variadic, it's a bit unusual

Reply:
Yes